pkg/cmd/api/api.go

```diff
@@ -281,8 +281,8 @@ func apiRun(opts *ApiOptions) error {
 		}
 	}
 
-	bodyWriter := opts.IO.Out
-	headersWriter := opts.IO.Out
+	var bodyWriter io.Writer = opts.IO.Out
+	var headersWriter io.Writer = opts.IO.Out
```

Comment:
Is this necessary because the Go compiler not recognize that the `fileWriter` interface implements the `io.Writer` interface?

Reply:
No, it's because without an explicit type, `bodyWriter` would become the exact same type as `IOStreams.Out`, which is `fileWriter`. Then, later in this function I want to reassign `bodyWriter` to another Writer which doesn't necessarily have the `Fd()` method which isn't needed.

Basically, I'm just narrowing down the interface for the purpose of the rest of the code in this function.